src/react/ReactQueryConfigProvider.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react'
 import { DEFAULT_CONFIG, defaultConfigRef } from '../core/config'
+import { useQueryCache } from './ReactQueryCacheProvider'
 
 //
 
 const configContext = React.createContext()
 
 export function useConfigContext() {
-  return React.useContext(configContext) || defaultConfigRef.current
+  const queryCache = useQueryCache()
+  return React.useContext(configContext) || queryCache.configRef.current
```

Comment:
The default `queryCache.configRef` will be `defaultConfigRef` out-of-the-box unless you're using `makeQueryCache`.